### PR TITLE
ROE-1693 Display second nationality on check your answers page

### PR DESCRIPTION
**Pull request by mattch1**
### JIRA link

https://companieshouse.atlassian.net/browse/ROE-1693

### Change description

Split summary list to allow optional rows.

### Work checklist

- [x] Tests added where applicable
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: test/controllers/check.your.answers.controller.spec.ts [hunk 1]
```diff
@@ -71,7 +75,11 @@ import {
   BO_GOV_ID_URL,
   MO_IND_ID_URL,
   MO_CORP_ID_URL,
-  TRANSACTION_ID, PUBLIC_REGISTER_NAME, PUBLIC_REGISTER_JURISDICTION, REGISTRATION_NUMBER,
+  TRANSACTION_ID,
+  PUBLIC_REGISTER_NAME,
+  PUBLIC_REGISTER_JURISDICTION,
+  REGISTRATION_NUMBER,
+  BENEFICIAL_OWNER_INDIVIDUAL_OBJECT_MOCK, MANAGING_OFFICER_OBJECT_MOCK,
```
Comment (markpit) [inline test/controllers/check.your.answers.controller.spec.ts:82]: I'd have a new line after the comma so MANAGING_OFFICER_OBJECT_MOCK is on a new line

File: views/includes/check-your-answers/beneficial-owner-individual.html [hunk 2]
```diff
@@ -46,145 +46,173 @@
   {% endif %}
 {% endset %}
 
+{#  #}
```
Comment (markpit) [inline views/includes/check-your-answers/beneficial-owner-individual.html:49]: The {# #} are empty comments here, so i would add a comment just to highlight what is happening,
maybe 
`{# Build the rows array to be used by the govukSummaryList #}`

Comment (mattch1) [reply views/includes/check-your-answers/beneficial-owner-individual.html]: Extended this comment to mention optional rows.

File: test/controllers/check.your.answers.controller.spec.ts (hunk 1 above)
Comment (mattch1) [reply test/controllers/check.your.answers.controller.spec.ts]: New line

**Comment by markpit** [comment]
Looks ok to me, will ask @mouhajer-ch if he is ok with the row building solution first before i approve (as I suggested this solution, so would be good to get Moh's input)

**Review by mouhajer-ch — APPROVED** [review]

**Pull request merged**